docs/tutorial/examples.md

```diff
@@ -50,7 +50,7 @@ guide!).
 
 You can find the full list of "How to?" in the sidebar. If there is
 something that you would like to do that is not documented, please join
-our [Discord server][] and let us know!
+our [Discord server][discord] and let us know!
 
-[discord server]: https://discord.com/invite/electron
+[discord]: https://discord.com/invite/APGC3k5yaH
```

Comment:
Sorry, super nitpicky, but I think we use `https://discord.com/invite/electronjs` - just think we should keep that in case this one expires 🙂